### ISSUE TITLE
Missing MASVS references

### ISSUE THREAD

**Issue by commjoen** [OWASP/owasp-mstg#1195]
We are missing references to :
- [x] V1.X of the MASVS: these still need to be added somewhere? More specifically still to do:

- [x] V1.8: "There is an explicit policy for how cryptographic keys (if any) are managed, and the lifecycle of cryptographic keys is enforced. Ideally, follow a key management standard such as NIST SP 800-57."
     - NIST SP 800-57 Part 1 Rev. 4: https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-4/final

- [x] V1.9: "A mechanism for enforcing updates of the mobile app exists."

- [x] V3.5 of the MASVS for iOS: The app doesn't re-use the same cryptographic key for multiple purposes.
- [x] v5.6 of the MASVS for iOS (covered in #1199)
- [x] V6.4 (covered in #1184)
- [x] V8.2-8.6 (covered in #1116)

**Comment by commjoen** [comment]
Can be closed after #1200 is merged.

**Comment by commjoen** [comment]
References complete, still have to fix #1199, but it has its own issue, and #1116 will need its own attention
